### PR TITLE
[systemtest] Update `SetupDrainCleaner` class after update to `0.4.0`

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lukywill16@gmail.com>

### Type of change

- Bugfix

### Description

This PR adds `Role` and `RoleBinding` handling to `SetupDrainCleaner` class after update to `0.4.0`. Because new files were added, our installation method was failing.

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/main/java/io/strimzi/systemtest/resources/draincleaner/SetupDrainCleaner.java [hunk 1]
```diff
@@ -68,6 +72,22 @@ public void applyInstallFiles(ExtensionContext extensionContext) {
...
+                            .endMetadata()
+                            .build());
+                        break;
+                    case Constants.ROLE_BINDING:
+                        RoleBinding roleBinding = TestUtils.configFromYaml(file, RoleBinding.class);
+                        ResourceManager.getInstance().createResource(extensionContext, new RoleBindingBuilder(roleBinding)
+                            .editMetadata()
+                                .withNamespace(Constants.DRAIN_CLEANER_NAMESPACE)
+                            .endMetadata()
+                            .build());
```
[the remark this answers is not in the document]
Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/draincleaner/SetupDrainCleaner.java:89]: good point

[the remark this answers is not in the document]
Comment (scholzj) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/draincleaner/SetupDrainCleaner.java]: Sorry, maybe I just don't understand how exactly you use it in the STs ... but how does that change the namespace? You now don't change the namespace at all.

[the remark this answers is not in the document]
Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/draincleaner/SetupDrainCleaner.java]: Yeah, sorry.
I misread the comment 🤦

**Review by scholzj — APPROVED** [review]

**Review by Frawless — APPROVED** [review]

**Pull request merged**